source/Halibut.Tests/CancellationViaClientProxyFixture.cs

```diff
@@ -41,7 +42,7 @@ await using (var clientAndService = await clientAndServiceTestCase.CreateTestCas
                 tokenSourceToCancel.CancelAfter(TimeSpan.FromMilliseconds(100));
 
                 (await AssertAsync.Throws<Exception>(() => echo.IncrementAsync(halibutRequestOption)))
-                    .And.Should().Match(x => x is ConnectingRequestCancelledException || (x is HalibutClientException && x.As<HalibutClientException>().Message.Contains("The Request was cancelled while Connecting")));
```

Comment:
In theory these messages are important for backwards comparability, are we missing tests?